tests/Backoffice/Sales/Clients/Application/Create/ClientCreatorTest.php

```
@@ -0,0 +1,62 @@
+        $clientRepository->shouldReceive('nextCode')
+            ->once()
+            ->andReturn($client->code());
+
+        $clientRepository->shouldReceive('save')
+            ->once()
+            ->with($this->similarTo($client))
+            ->andReturnNull();
+
+        $eventBus = $this->eventBus();
+        $event = new ClientCreatedDomainEvent(
+            $client->id(),
```

PHPStan (2.0.3)

Parameter #1 $aggregateId of class MedineTech\Backoffice\Sales\Clients\Domain\ClientCreatedDomainEvent constructor expects string, string|null given.


(argument.type)